src/signalingclient/DefaultSignalingClient.ts

```diff
@@ -107,13 +109,31 @@ export default class DefaultSignalingClient implements SignalingClient {
     joinFrame.clientDetails = SdkClientDetails.create(sdkClientDetails);
     joinFrame.audioSessionId = this.audioSessionId;
     joinFrame.wantsCompressedSdp = DefaultSignalingClient.CLIENT_SUPPORTS_COMPRESSION;
-
+    joinFrame.desiredServerSideNetworkAdaption = this.convertServerSideNetworkAdaptionEnumToSignaled(
+      settings.desiredServerSideNetworkAdaption
+    );
+    joinFrame.serverSideNetworkAdaptionOverrideSupport = settings.serverSideNetworkAdaptionOverrideSupport.map(
+      this.convertServerSideNetworkAdaptionEnumToSignaled
+    );
     const message = SdkSignalFrame.create();
     message.type = SdkSignalFrame.Type.JOIN;
     message.join = joinFrame;
     this.sendMessage(message);
   }
 
+  private convertServerSideNetworkAdaptionEnumToSignaled(
```

Comment:
nit: Seem like this should be a helper function somewhere

Reply:
moved!